.env.example

```diff
@@ -9,5 +10,9 @@ SMTP_LOGIN=jurchenko.a@gmail.com
 SMTP_PASS=sfHXGMTUjxZ7PObc
 SMTP_SERVER=smtp-relay.sendinblue.com
 EMAIL_SENDER=admin@medon.com
-JWT_SECRET=my_secret
-CONFIRMATION_TOKEN_EXPIRED_AT=2h
\ No newline at end of file
+JWT_SECRET=my_secret_auth
+CONFIRMATION_TOKEN_EXPIRED_AT=2h
+JWT_EXPIRATION_TIME=2h
+GOOGLE_CLIENT_ID=790559126715-933s8ngov6rqllmsssp6jj6co9sqe9s2.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=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
+GOOGLE_CALLBACK_URL=http://localhost:3333/auth/google/redirect
```

Comment:
its example, it should not have real data

Reply:
It was for guys) so they can test it:) Removed